lib/workflow/client.js

```diff
@@ -129,6 +129,10 @@ class Workflow {
     };
   }
 
+  profileTasks({ profileId }) {
```

Comment:
There's some inconsistency between this and [openTasks](https://github.com/UKHomeOffice/asl-service/blob/master/lib/workflow/client.js#L136) which takes a plain id as an argument rather than wrapping in an object.

```suggestion
  profileTasks(profileId) {
```